language-design.md

```diff
@@ -47,7 +85,9 @@ In what way is an API a language?
 
 **Response**
 
-
+APIs differ from DSLs in that APIs don't implement their own syntax, and instead borrow it
+from the host language, however they still have vocabulary and grammar. You still must use 
+the vocabulary of the API in the way that it allows.
```

Comment:
I agree with Alex. On a related note, what do you mean by APIs not implementing their own syntax, but still having their own grammar? Isn't grammar a part of syntax?